eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -69,6 +69,38 @@
                 }
             }
         },
+        {
+            "StaticConfigs": {
+                "Windows2019": {
+                    "OSVmImage": "MMS2019",
+                    "Pool": "azsdk-pool-mms-win-2019-general",
+                    "BuildArgs": "--parallel 8",
+                    "CMAKE_GENERATOR": "Visual Studio 16 2019",
+                    "CmakeArgs": " -DBUILD_TRANSPORT_WINHTTP=ON ",
+                    "PublishMapFiles": "true"
+                }
+            },
+            "TargetPlatform": {
+                "UWP_debug": {
+                    "CMAKE_SYSTEM_NAME": "WindowsStore",
```

Comment:
`[CMAKE_SYSTEM_NAME](https://cmake.org/cmake/help/v3.16/variable/CMAKE_SYSTEM_NAME.html)` and `[CMAKE_SYSTEM_VERSION](https://cmake.org/cmake/help/v3.16/variable/CMAKE_SYSTEM_VERSION.html)` are not environment variables and setting these does not result in correct UWP builds. This is being addressed in https://github.com/Azure/azure-sdk-for-cpp/pull/3263